python_lkvm/lkvm.py

```diff
@@ -482,3 +482,31 @@ def sandbox(self, cpus, mem, shmem, console, kernel, params,
 
     def is_supported(self):
         return os.path.isfile(LKVM_PATH)
+
+    def find_root_partition(self, image_filename):
+        partition = None
+        command = ["fdisk", str(image_filename), "-l"]
+        _PIPE = subprocess.PIPE
+        obj = subprocess.Popen(command,
```

Comment:
I think, we could modify this function https://github.com/clearlinux/python-lkvm/blob/master/python_lkvm/lkvm.py#L66-L110 instead of re-implementing the subprocess.Popen